src/components/modals/email-confirm-modal/email-confirm-modal.js

```diff
@@ -0,0 +1,73 @@
+import { Button, Dialog, IconButton } from '@mui/material'
```

Comment:
please check if im doing it right

i have created 2 templates for modals email confirmation or rejection